src/core/src/preprocess/preprocess_steps_impl.cpp

```diff
@@ -13,6 +13,7 @@
 #include "openvino/opsets/opset8.hpp"
 #include "openvino/util/common_util.hpp"
 #include "transformations/rt_info/preprocessing_attribute.hpp"
+#include "openvino/opsets/opset1.hpp" 
```

Comment:
Is it required?

Reply:
yeah I have used it in.

Reply:
After changes it can be removed is not used at all.
The clamp operator is also in opset8 (same version) and opset8 included already.